### DISCUSSION TITLE
Installation Problems (AMD 5700xt)

### DISCUSSION THREAD

**Discussion by LostAndDead** [lshqqytiger/stable-diffusion-webui-directml#536] [Q&A]
So im trying to install this on my 5700xt (50th Aniversary Edition) card, I have followed the relevant guide listed here
https://github.com/CS1o/Stable-Diffusion-Info/wiki/Installation-Guides#amd-automatic1111-with-zluda

Everything installs just fine, added all the PATH variables and restarting after everything whenever I try and run it I get this error:
`rocBLAS error: Could not load C:\Program Files\AMD\ROCm\6.1\bin\/rocblas/library/TensileLibrary.dat`

This comes after the line saying its launching the webui
`Launching Web UI with arguments: --use-zluda --update-check --skip-ort`

Drivers is the latest available and I believe I have followed the guide correctly.

Any ideas on how to fix it? Seems like a simple path error, the file it's referring too does exist, just the path is wrong/broken, the true path is:
`C:\Program Files\AMD\ROCm\6.1\bin\rocblas\library\TensileLibrary.dat`

Seems the `\/` is breaking it but idk how to fix it.

**Comment by CS1o** [comment]
Hey, my guide says to use Rocm 5.7 for the RX5700XT and also links the requires rocm library files, You installed HIP SDK 6.1 which means you either need to uninstall that and use 5.7 or you have to download the compatible Rocm library files for you card for 6.1.
You can find them here:
https://github.com/likelovewant/ROCmLibs-for-gfx1103-AMD780M-APU/releases/download/v0.6.1.2/rocm.gfx1010-xnack-.for.hip.sdk.6.1.2.7z
Then proceed with my Guide at step 3.3

If they wont work, then you need to go the 5.7 route.

**Comment by LostAndDead** [comment]
Thats my mistake, its working now. While images generate fine if I want to use Highres. fix it takes ages to finish, I'm talking 20-40 mins per image, I also can't even seem to interrupt it, just does nothing then eventually says connection timed out.

**Reply by CS1o** [reply]
> Thats my mistake, its working now. While images generate fine if I want to use Highres. fix it takes ages to finish, I'm talking 20-40 mins per image, I also can't even seem to interrupt it, just does nothing then eventually says connection timed out.

Alright, make sure you when upscaling to set the Hires Steps always to 10 so it doesnt run out of vram or takes ages.
Also you need the Tiled VAE extension. That can help alot for upscaling to not run out of vram at higher resolution.

**Comment by LostAndDead** [comment]
Doesnt seem to have made much difference. According to the console its done at 100% but just sits there visually doing nothing for ages.
Just sits like this for a while, is really still doing something tho as my GPU is maxing out
![WindowsTerminal_PElW1VFshC](https://github.com/user-attachments/assets/74200088-6be7-40e3-b798-07cb1ac38daa)

**Reply by ride5k** [reply]
there is a discrepancy with the way the interface is calculating iterations that gives a false impression of ridiculously high generation rates, only to take extra time at the end.

**Reply by LostAndDead** [reply]
So the console output is just wrong? I can get the webui being wrong but the console?

**Reply by CS1o** [reply]
console output looks normal.
Which model and resolution do you use?
Also make sure you dont have any games or Wallpaper Engine running in the background.

**Reply by ride5k** [reply]
> So the console output is just wrong? I can get the webui being wrong but the console?

see https://github.com/lshqqytiger/stable-diffusion-webui-amdgpu/issues/502

**Reply by LostAndDead** [reply]
Even after the console says 100%it sits on that for about 2 minutes till its done, this is meinamix V11 at 512x768, no high res. DPM++ SDE KARRAS, 25 steps. No games are running but wallpaper engine was. Quitting it seems to make little to no measurable difference. Enabling the tiled VAE/Diffusion does nothing, says the input size is tiny and unnecessary to tile

**Reply by CS1o** [reply]
Can you try using an sdxl model like JuggernautXL at 1024x1024 with DPM++ 2M, 30 steps, to see if there is the same issue?

**Reply by LostAndDead** [reply]
Tried the exact model and parameters you said with Tiled VAE + Diffusion, it took 26 mins, about 5 mins of that it was sat on 96% on the webui, the console was doing something else. Over all the eta was much better tracked. The image is also completely broken, thought it might be the tile diffusion so I tried again with just tiled VAE, it took only 9 mins and no real empty time and a decent image

**Reply by CS1o** [reply]
Yea dont use Tiled Diffusion, only use Tiled VAE when upscaling or generating at high res.
Also add --medvram-sdxl to the webui-user.bat. that can help with the speed for sdxl models.

**Reply by LostAndDead** [reply]
Been a bit as I was away but any idea how to speed up the generation of smaller images outside of sdxl, such as the model I was using previously? If that's just how long it takes then fine but any ideas on if It should be faster, I have no benchmark really
